<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>supprime</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_client">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>id_client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom_client">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nom_client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom_client">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>prenom_client</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>adresse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_tel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>num_tel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>11</y>
      <width>981</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
